django/cantusdb_project/main_app/templates/chant_create.html

```diff
@@ -282,9 +282,13 @@ <h5><a id="source" href="{% url 'source-detail' source.id %}">{{ source.siglum }
                                         document.getElementById('id_cantus_id').value = "{{ suggestion.cantus_id }}";
                                         {% comment %} document.getElementById('id_genre').value = {{ suggestion.genre_id }}; {% endcomment %}
                                         document.getElementById('id_manuscript_full_text_std_spelling').value = "{{ suggestion.fulltext }}";
-                                        var genre_id = {{ suggestion.genre_id }};
+                                        var genre_id = {{ suggestion.genre_id | default_if_none:"null" }};
                                         var genre_name = "{{ suggestion.genre_name }}";
 
+                                        // in case suggestion.genre_id is None, don't try to change the #id_genre selector
+                                        if (genre_id == null) {
```

Comment:
Mmmmm. Python writing JavaScript. My favourite. 

You probably want `===` here.

Reply:
The question of `===` vs. `==` was on my mind while writing this. I think someone in a StackOverflow discussion said that `==` was correct, since professional JS developers will understand the difference between the two or some similar justification, but looking at it now, I'm inclined to agree with you. Let me make this change - thanks.